cmd/activator/main.go

```diff
@@ -195,9 +199,11 @@ func main() {
 	concurrencyReporter := activatorhandler.NewConcurrencyReporter(ctx, env.PodName, statCh)
 	go concurrencyReporter.Run(ctx.Done())
 
+	proxyTransport := pkgnet.NewAutoTransport(*maxIdleProxyConns, 100 /*maxidleperhost*/)
```

Comment:
Do we want to tweak maxidleperhost as well?

Reply:
actually, probably, Ill add

Reply:
done - thanks!